docs/index.md

```diff
@@ -0,0 +1,33 @@
+Welcome to LibreTime
+====================
+
+LibreTime makes it easy to run your own online radio station. Check out some [features](features.md) and [screenshots](features.md#screenshots), then [install it](install.md) and start broadcasting!
+
+LibreTime is Free/Libre and Open Source Software (FLOSS). Among other things, this means that you have the freedom to:
+
+* Run it royalty-free for as long as you like.
+* Read and alter the code that makes it work (or hire someone to do this for you!)
+* Contribute documentation, bug-fixes, etc. so that everyone in the community benefits.
+
+LibreTime is a fork of AirTime due to stalled development of the FLOSS version. For background on this, see this [open letter to the Airtime community](https://gist.github.com/hairmare/8c03b69c9accc90cfe31fd7e77c3b07d).
+
+
+Getting Started
+---------------
+
+The easiest way to check out LibreTime for yourself is to run a local instance in a virtual machine:
+
+1. Install Git, Vagrant and Virtualbox:
+```console
+$ sudo apt-get install git vagrant virtualbox
+```
+2. Clone this repository locally:
+```console
+$ git clone https://github.com/libretime/libretime.git
+```
+3. Launch the VM:
+```console
+$ cd libretime
+$ vagrant up
```

Comment:
Wouldn't this need a `Vagrantfile`? I might have an old one collecting dust somewhere, personally I mostly use docker nowadays.